website/content/en/status/report-2023-10-2023-12/gcc.adoc

```diff
@@ -0,0 +1,24 @@
+=== GCC on FreeBSD
+
+Links: +
+link:https://gcc.gnu.org/[GCC Project] URL: link:https://gcc.gnu.org/[] +
+link:https://gcc.gnu.org/gcc-10/[GCC 10 release series] URL: link:https://gcc.gnu.org/gcc-10/[] +
+link:https://gcc.gnu.org/gcc-11/[GCC 11 release series] URL: link:https://gcc.gnu.org/gcc-11/[] +
+link:https://gcc.gnu.org/gcc-12/[GCC 12 release series] URL: link:https://gcc.gnu.org/gcc-12/[] +
+link:https://gcc.gnu.org/gcc-13/[GCC 13 release series] URL: link:https://gcc.gnu.org/gcc-13/[]
+
+Contact: Lorenzo Salvadore <salvadore@FreeBSD.org>
+
+The link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=273397[process] to update GCC default version to version 13 is getting forward.
```

Comment:
Suggestion for accessibility:

link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=273397[Updating GCC default version to 13] is moving ahead.